### PR TITLE
bugherd task #548

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/reference/smartrest-one-bundle/the-protocol.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Each row yielded by the SmartREST endpoint represents a set of extracted values
 
 The CSV (comma-separated values) format is used for communication with the SmartREST endpoint. The following rules must be followed to ensure a frictionless communication.
 
-* Every row must be terminated by the `\n` character sequence.
+* Every linebreak must be encoded as by the character sequence `\n`.
```
Comment (BeateRixen) [inline content/reference/smartrest-one-bundle/the-protocol.md:37]: "as by" seems to be wrong?

Comment (rssag) [reply content/reference/smartrest-one-bundle/the-protocol.md:37]: oh, yes, typo. should only be "by"

Comment (BeateRixen) [reply content/reference/smartrest-one-bundle/the-protocol.md:37]: @rssag Did you commit your change?

**Review by BeateRixen — APPROVED** [review]

**Comment by rssag** [comment]
fixed in 10.11.0 and 10.13.0

**Pull request merged**